src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -4,7 +4,7 @@
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
 
     <!-- NuGet -->
-    <Version>1.3.1</Version>
+    <Version>1.3.2</Version>
```

Comment:
Did you need this? Usually we bump the version up everywhere in the solution just before doing a new release. 

There's many more places to update than these 3 projects. For example, [the clr profiler will still inject references](https://github.com/DataDog/dd-trace-dotnet/blob/0f25feb4698bfe4dfb5cc4f219fa8a269a4b4fee/src/Datadog.Trace.ClrProfiler.Native/version.h#L3) to `Datadog.Trace.ClrProfiler.Managed` 1.3.1, so this change breaks automatic instrumentation unless you change the version there as well.

Also, since we're adding a new feature, we should go to `1.4` according to semantic versioning.

Reply:
I ran into issues because I had the .NET Tracer 1.3 installed which added `Datadog.Tracer 1.2.0.0` into the GAC and then subsequent tests tried to use that version. After removing the product from my machine this was no longer an issue. We can wait to do version changes until we're ready to release, we just have to be mindful of this in case we need to ship a bug fix before the next minor version.